datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java

```diff
@@ -131,7 +132,7 @@ private List<CDPStructuredEvent> retrievePagedCloudbreakServiceEvents(SdxCluster
             List<CloudbreakEventV4Response> cloudbreakEventV4Responses = ThreadBasedUserCrnProvider.doAsInternalActor(
                     regionAwareInternalCrnGeneratorFactory.iam().getInternalCrnForServiceAsString(),
                     () ->
-                            eventV4Endpoint.getPagedCloudbreakEventListByCrn(sdxCluster.getCrn(), page, size, false));
+                            eventV4Endpoint.getPagedCloudbreakEventListByCrn(getCloudbreakCrn(sdxCluster), page, size, false));
             return cloudbreakEventV4Responses.stream().map(entry -> convert(entry, sdxCluster.getCrn())).collect(toList());
```

Comment:
sanity check: in the response the DL crn should be used even if the stack crn is different, right?

Reply:
i guess, since this is logic in dl service